paymentrequest/credit-cards/demo.js

```diff
@@ -8,8 +8,6 @@ function initPaymentRequest() {
       'visa', 'mir'];
   let types = ['debit', 'credit', 'prepaid'];
   let supportedInstruments = [{
-    supportedMethods: networks,
-  }, {
     supportedMethods: ['basic-card'],
```

Comment:
Could you also please change `['basic-card']` array to `'basic-card'` string? Thank you.